### ISSUE TITLE
[Paper Note] Video PreTraining (VPT): Learning to Act by Watching Unlabeled Online Videos, Bowen Baker+, arXiv'22, 2022.06

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#4707]
# URL
- https://arxiv.org/abs/2206.11795
# Authors
  - Bowen Baker
  - Ilge Akkaya
  - Peter Zhokhov
  - Joost Huizinga
  - Jie Tang
  - Adrien Ecoffet
  - Brandon Houghton
  - Raul Sampedro
  - Jeff Clune
# Abstract
  - Pretraining on noisy, internet-scale datasets has been heavily studied as a technique for training models with broad, general capabilities for text, images, and other modalities. However, for many sequential decision domains such as robotics, video games, and computer use, publicly available data does not contain the labels required to train behavioral priors in the same way. We extend the internet-scale pretraining paradigm to sequential decision domains through semi-supervised imitation learning wherein agents learn to act by watching online unlabeled videos. Specifically, we show that with a small amount of labeled data we can train an inverse dynamics model accurate enough to label a huge unlabeled source of online data -- here, online videos of people playing Minecraft -- from which we can then train a general behavioral prior. Despite using the native human interface (mouse and keyboard at 20Hz), we show that this behavioral prior has nontrivial zero-shot capabilities and that it can be fine-tuned, with both imitation learning and reinforcement learning, to hard-exploration tasks that are impossible to learn from scratch via reinforcement learning. For many tasks our models exhibit human-level performance, and we are the first to report computer agents that can craft diamond tools, which can take proficient humans upwards of 20 minutes (24,000 environment actions) of gameplay to accomplish.
# Translation (by gpt-5-nano)
- ノイズを含んだインターネット規模のデータセットでの事前学習は、テキスト、画像、その他のモダリティに対して広範で一般的な能力を持つモデルを訓練する手法として広く研究されてきた。しかし、ロボティクス、ビデオゲーム、コンピュータの使用といった多くの逐次決定領域では、公開データには同じ方法で行動事前知識を訓練するのに必要なラベルが含まれていない。私たちは、逐次決定領域にもインターネット規模の事前学習パラダイムを拡張するため、オンラインのラベルなし動画を観察して行動を学ぶ半教師付き模倣学習を導入する。具体的には、少量のラベル付きデータで、巨大なラベルなしオンラインデータ源（ここではMinecraftをプレイする人々のオンライン動画）にラベルを付けるのに十分な精度の逆ダイナミクスモデルを訓練し、それを用いて一般的な行動事前知識を訓練することを示す。ネイティブな人間インターフェース（20Hzのマウスとキーボード）を使用しているにもかかわらず、この行動事前知識は非自明なゼロショット能力を持ち、模倣学習と強化学習の双方でファインチューニングすることで、強化学習からゼロから学ぶことが不可能な難しい探索課題へ適用できることを示す。多くのタスクにおいて私たちのモデルは人間レベルの性能を示し、ダイヤモンドの道具を作成できるコンピュータエージェントを報告したのは初めてのことであり、それは熟練した人間でさえ20分以上（環境アクション24,000回）を要して達成する。
# Summary (by gpt-4o-mini)
- オンラインのラベルなし動画を用いた半教師付き模倣学習により、逐次決定領域へインターネット規模の事前学習を拡張。逆ダイナミクスモデルを利用して、少量のラベル付きデータから一般的な行動知識を獲得。これにより、模倣学習と強化学習でのファインチューニングが可能となり、困難な探索課題に対するゼロショット能力を示す。初めて、ダイヤモンドの道具を作成できるコンピュータエージェントが報告され、人間レベルの性能を実現。

**Comment by AkihikoWatanabe** [comment]
Inverse Dynamics Model (IDM)

**Comment by AkihikoWatanabe** [comment]
observationによる状態の変化が、どのアクションによって引き起こされたかを推定するモデル（Inverse Dynamics Model）を学習し（すべてのobservationのtrajectoryから時刻tでのアクションを予測するモデル）アクション-状態遷移のダイナミクスの知識を理解したモデルを学習。その後大量のunlabeled dataに対してIDM Modelによって、アクションをラベル付けし、当該アクションラベルを教師として、Forward Dynamics Model （時刻tまでの状態遷移が与えられた時に次にどのようなアクションが実施されるかを予測するモデル）を学習することで事前学習をする手法な模様。
